src/signature/SigningAccount.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+// Copyright 2017 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "xdr/Stellar-SCP.h"
+#include "xdr/Stellar-ledger.h"
+
+namespace stellar
+{
+
+class AccountFrame;
+
+struct SigningAccount
+{
+    AccountID accountID;
```

Comment:
our code base follows the convention that members are named starting with `m` (lower case m)